app/models/glossary_mapping.rb

```diff
@@ -0,0 +1,35 @@
+class GlossaryMapping < ApplicationRecord
+    include SpotlightSearch
+    include Paginator
+
+      validates :consultation_id, presence: true
+      validates :glossary_id, presence: true
+
+    belongs_to :consultation
+    belongs_to :wordindex, foreign_key: "glossary_id"
+    
+    
+    scope :alphabetical, -> { order(consultation_id: :asc) }
```

Comment:
The scope says alphabetical, but it seems to be sorting by the ID

Reply:
changed the variable name from alphabetical to order_by_id.